### PR TITLE
add typing for ownca/utils

### LINKED ISSUE #62: Implement typing to ``ownca/utils.py``

### PR DESCRIPTION
**Pull request by kairoaraujo**
This commit is part of the plan to add to the ownca typing.
It adds to the ownca typing to the ``ownca/utils`` and its usage accross
the codebase and tests.

Closes #62

Signed-off-by: Kairo de Araujo <kairo@kairo.eti.br>

### CODE REVIEW

File: ownca/utils.py [hunk 1]
```diff
@@ -53,12 +65,14 @@ def file_data_status(ca_status):
...
 
-def _create_ownca_dir(ownca_dir):
+def _create_ownca_dir(ownca_dir: str) -> None:
     """
     Creates the CA directory.
 
-    :param ownca_dir: :string: full path directory for ownca
+    :param ownca_dir: full path directory for ownca
+    :type ownca_dir: string, required
     :return: bool
```
Comment (lawndoc) [inline ownca/utils.py:74]: The return type in the function declaration (None) doesn't match the return type in the docstring (bool). It appears that this function has no return statement, so we should change the docstring.

File: Pipfile [hunk 2]
```diff
@@ -11,7 +11,11 @@ flake8 = "*"
...
 sphinx = "*"
 sphinx-rtd-theme = "*"
+twine = "*"
+mypy = "*"
+isort = "*"
 
 [packages]
 cryptography = "<35.0.0,>=3.4.6"
 voluptuous = "*"
+black = "==21.12b0"
```
Comment (lawndoc) [inline Pipfile:21]: Should this be under `dev-packages` or is this needed for the runtime? I haven't used black before.

**Review by lawndoc — COMMENTED** [review]
Just a couple of things I'd like you to take a second look at. Nice work!

File: ownca/utils.py (hunk 1 above)
Comment (kairoaraujo) [reply ownca/utils.py:74]: Nice catch

File: Pipfile (hunk 2 above)
Comment (kairoaraujo) [reply Pipfile:21]: Yes. You're right. It is only for a development environment.

**Review by lawndoc — APPROVED** [review]

**Comment by kairoaraujo** [comment]
Thanks for your review @lawndoc

**Pull request merged**